### ISSUE TITLE
After upgrading to Angular7 mat-paginator tooltip throws error in console

### ISSUE THREAD

**Issue by larisaVasile** [angular/components#14097]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Do not throw error.

#### What is the current behavior?

TooltipComponent.html:1 ERROR TypeError: Cannot read property 'matches' of null
    at Object.eval [as updateRenderer] (TooltipComponent.html:1)
    at Object.debugUpdateRenderer [as updateRenderer] (core.js:20458)
    at checkAndUpdateView (core.js:19833)
    at callViewAction (core.js:20069)
    at execComponentViewsAction (core.js:20011)
    at checkAndUpdateView (core.js:19834)
    at callViewAction (core.js:20069)
    at execEmbeddedViewsAction (core.js:20032)
    at checkAndUpdateView (core.js:19829)
    at callViewAction (core.js:20069)
#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>

**Comment by ovidiugiorgi** [comment]
I tried reproducing the issue on [StackBlitz](https://stackblitz.com/edit/angular-material2-issue-qeewgt), but it seems to work fine. You can see under dependencies that it uses angular 7 packages.

**Comment by ouassimBenMosbah** [comment]
I faced the same issue. I was able to reprpoduce it on the stackblitz below
[stackBlitz](https://stackblitz.com/edit/angular-material2-issue-8iilzp)

I had to use the version 6.4.7 of @angular/material to reproduce this bug

**Comment by jotatoledo** [comment]
@ouassimBenMosbah your dependencies are wrong, material is outdated in your blitz

**Comment by josephperrott** [comment]
Can you provide a reproduction in [StackBlitz]( https://goo.gl/wwnhMV).  This will help us to investigate the issue you are seeing.

**Comment by r0maks** [comment]
Here's a Stack overflow thread of the issue described: https://stackoverflow.com/questions/53434382/angular-material-2-tooltip-issue

I too am having this issue. Error gets spit out onto the console when hovering over a tooltip after updating angular cdk to 7.1.1 from 6.4.6

**Comment by ikismail** [comment]
Facing the same issue while updating @angular/cdk from **6.4.6** to **7.2.0**
any solutions to get rid of this issue? 
I can't just stay with 6.4.6 version cause I need to use **drag and drop module**.

**Comment by cyberbeast** [comment]
Is there a fix for this in the works? 

`ERROR TypeError: Cannot read property 'matches' of null`

**Comment by houssemFat-DDF** [comment]
Same issue when i do hover the next page button 
```
 <mat-paginator
        [pageSize]="pageLimit"
        [pageIndex]="currentPageIndex"
        [pageSizeOptions]="[5, 10, 25, 100]"
        [length]="list.length"
        (page)="updatePagination($event)">
      </mat-paginator>
```
*Package json dependencies *
```
"dependencies": {
    "@angular/animations": "^6.1.10",
    "@angular/cdk": "^7.3.1",
    "@angular/common": "^6.1.0",
    "@angular/compiler": "^6.1.0",
    "@angular/core": "^6.1.0",
    "@angular/flex-layout": "6.0.0-beta.17",
    "@angular/forms": "^6.1.0",
    "@angular/http": "^6.1.0",
    "@angular/material": "^6.1.0",
    "@angular/platform-browser": "^6.1.0",
    "@angular/platform-browser-dynamic": "^6.1.0",
    "@angular/router": "^6.1.0",
    "@ngx-translate/core": "^9.1.1",
    "@ngx-translate/http-loader": "^2.0.1",
    "classlist.js": "^1.1.20150312",
    "core-js": "^2.5.4",
    "moment": "^2.22.2",
    "rxjs": "6.1.0",
    "rxjs-compat": "6.1.0",
    "sortablejs": "^1.8.1",
    "web-animations-js": "^2.3.1",
    "zone.js": "^0.8.26"
  },
```

![image](https://user-images.githubusercontent.com/34309592/52642765-eec34180-2edb-11e9-845e-7c1f0567a852.png)

**Comment by asheikabrar** [comment]
same error any fix

**Comment by ShemaPatrick** [comment]
Same here!

**Comment by NeWbLt123** [comment]
Upgrading `@angular/material` to `7.3.3` fixed the problem for me.

**Comment by mritykum** [comment]
Upgraded to@angular/material to 7.3.3  not fixed the problem for me.

**Comment by nick-beers** [comment]
I found somewhere online that `@angular/material` and `@angular/cdk` should be always pulling similar versions. Originally my cdk was using ^7.3.7 as the version and material was using ~7.3.7, so I made them both use ^7.3.7, re-pulled node_modules and everything appeared to resolve.  It may be related to mismatching versions, but I didn't end up doing a full investigation into why that might have been my issue.
